src/store/wallet/actions.js

```diff
@@ -1,8 +1,24 @@
 import * as types from './types.js';
+import path from 'path'
+import { exec } from 'child_process'
+import util from 'util';
+const cmd = util.promisify(exec);
 
 const actions = {
-  [types.GET_NEW_MNEMONIC]() {
-    console.log('get new mnemonic');
+  async [types.GET_NEW_MNEMONIC] ({ commit , state}, { wordsNumber }) {
+     var wordsAmmountToBeGenerated=state.wordsNumbersAllowed.find(x => x === wordsNumber);
+     if(wordsAmmountToBeGenerated==null) throw 'not allowed lenght';     
+    
+     var cardanoPath= require('path').resolve('./');
+     const cardanoPath2 =  path.resolve(cardanoPath, 'cardano') ;
+     const addressCli = path.resolve('.', cardanoPath2, process.platform, 'cardano-address');
+     var cardano_words = `"${addressCli}" recovery-phrase generate --size ${wordsNumber}`;    
+    
+     const { stdout, stderr } = await cmd(cardano_words);
+     if(stderr) throw stderr;      
+    
+     var words =stdout.replace('\n', '');
+     commit(types.SET_MNEMONIC, words);       
```

Comment:
Alright so this isnt a viable decision. The only reason to move this 'type' operation to the Vue side is to support web/mobile capabilities, but we cannot use the binaries for such a move. We will need to remove the binary requirement and use this libary (most likely) https://github.com/vacuumlabs/cardano-crypto.js

Reply:
I see, don't marge it yet, I will use cardano-crypto.js there

Reply:
I will change line 12 to 20.